app/src/androidTest/java/mozilla/lockbox/Navigator.kt

```diff
@@ -98,18 +99,33 @@ class Navigator {
     fun gotoFingerprintOnboarding() {
         gotoFxALogin()
         fxaLogin { tapPlaceholderLogin() }
+        checkAtFingerprintOnboarding()
     }
 
     fun checkAtFingerprintOnboarding() {
         fingerprintOnboardingScreen { exists() }
     }
 
+    fun gotoAutofillOnboarding() {
+        gotoFxALogin()
+        fxaLogin { tapPlaceholderLogin() }
+        checkAtFingerprintOnboarding()
+        fingerprintOnboardingScreen { tapSkip() }
+        checkAtAutofillOnboarding()
+    }
+
+    fun checkAtAutofillOnboarding() {
+        autofillOnboardingScreen { exists() }
+    }
+
     fun gotoOnboardingConfirmation() {
         gotoFxALogin()
         fxaLogin { tapPlaceholderLogin() }
         if (FingerprintStore.shared.isFingerprintAuthAvailable) {
             checkAtFingerprintOnboarding()
             fingerprintOnboardingScreen { tapSkip() }
+            checkAtAutofillOnboarding()
```

Comment:
why is the autofillonboarding stuff in the `if (FingerprintStore.shared.isFingerprintAuthAvailable)` block? I'd rather see this moved out to its own check to `SettingStore.shared.isAutofillAvailable`.

When writing UI tests, I like to think about the way the code will work when run on different devices -- theoretically, these tests can be run in any environment, so they should be resilient to that.

Reply:
Good catch! I forgot to add in the setting store check